packages/repository/README.md

```diff
@@ -10,8 +10,9 @@ the legacy `loopback-datasource-juggler` and connector modules from LoopBack
 3.x.
 
 This module provides data access facilities to various databases and services.
-It contains the constructs for modeling and accessing data. Repositories can be
-used alone or as part of a `Controller` implementation.
+It contains the constructs for modeling and accessing data. We recommend using
+Repositories with implementation of `Controllers` but it can be used
```

Comment:
It'd be great if we have a sentence on what repositories are before we say `We recommend using Repositories...`